src/reducers/versions.spec.tsx

```diff
@@ -1774,4 +1778,203 @@ describe(__filename, () => {
       expect(state.versionInfo[beginFetchVersionId]).toEqual(undefined);
     });
   });
+
+  type TestHunkChange = {
+    lineNumber: number;
+    type: string;
+  };
+
+  type TestHunk = TestHunkChange[];
+
+  // This helper accepts an array of arrays, each of which is a list of changes
+  // which will be contained in a hunk.
+  const createDiff = (testHunks: TestHunk[]): DiffInfo => {
```

Comment:
`createFakeDiffWithChanges` is more verbose but I think it would help to make it clear that it's a simulated diff and that it only accepts changes (nothing else).